packages/dds/tree/api-report/tree.api.md

```diff
@@ -4,7 +4,10 @@
 
 ```ts
 
+import { FieldKind as FieldKind_2 } from './schemaTypes';
```

Comment:
What's with all the `_2`?

Reply:
The theory that @alexvy86 and I have is that these show up wherever type inference leads to dynamic imports in the d.ts. I think it's a bug somewhere in the toolchain but not sure where. I think @CraigMacomber looked into this at some point.